webhooks-extension/src/WebhookApp.js

```diff
@@ -105,7 +107,9 @@ const mapDispatchToProps = (dispatch, props) => ({
   fetchPipelines: namespace =>
     dispatch(props.actions.fetchPipelines({ namespace })),
   fetchServiceAccounts: namespace =>
-    dispatch(props.actions.fetchServiceAccounts({ namespace }))
+    dispatch(props.actions.fetchServiceAccounts({ namespace })),
+  fetchPipelineRuns: (namespace, filters) =>
+    dispatch(props.actions.fetchPipelineRuns(namespace, filters))
```

Comment:
see https://github.com/tektoncd/dashboard/blob/c1aa3e4238e2519731bf25f3ac019e9aa0b66e5b/src/actions/pipelineRuns.js#L27

`fetchPipelineRuns` expects an object `{ namespace, filters }`, I'm not sure how this is working in its current state. We probably need some additional tests if this is getting through.